pyprof/nvtx/config.py

```diff
@@ -26,4 +26,6 @@ def __new__(cls, *args, **kwargs):
             cls.func_stack_enabled = kwargs.get("enable_function_stack",
                                                 False) or kwargs.get("capture_input_ops", False)
             cls.capture_input_ops = kwargs.get("capture_input_ops", False)
+            cls.delay_graph_capture = kwargs.get("delay_graph_capture", False)
+            cls.debug_en            = kwargs.get("debug_en", False)
```

Comment:
Can we rename this flag to something like `debug_graph`? We don't yet have it but its a good idea to have separate debug flags for separate features.

Reply:
done debug_graph